controller/controller_server/exception_handler.py

```diff
@@ -11,6 +11,7 @@
 status_codes_by_exception = {
     ValidationException: grpc.StatusCode.INVALID_ARGUMENT,
     array_errors.IllegalObjectName: grpc.StatusCode.INVALID_ARGUMENT,
+    array_errors.IllegalObjectID: grpc.StatusCode.INVALID_ARGUMENT,
```

Comment:
How this is related to this PR?

Reply:
it an error that was added to the possible error that can be returned..

Reply:
Is it a common exception?

Reply:
just like IllegalObjectName